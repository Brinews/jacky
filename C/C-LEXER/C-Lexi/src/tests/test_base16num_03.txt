ABCx16

1x16
