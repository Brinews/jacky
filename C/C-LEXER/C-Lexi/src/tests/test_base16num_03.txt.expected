{ABCx16 , ID , 1}
{1x16 , BASE16NUM , 3}
{ , END_OF_FILE , 5}
